### ISSUE TITLE
FC layer

### ISSUE THREAD

**Issue by hezw2016** [XuecaiHu/Meta-SR-Pytorch#12]
Hi, Xuecai

Thank you for sharing this wonderful work!!!
I have a small question about the parameters in the FC layer. In 'metardn' model, the first FC layer is like: nn.Linear(3,256). But in 'metaedsr' model, the first FC layer is like: nn.Linear(6,256). I was confused about the parameter setting, I think both of them should be nn.Linear(3,256). Because according to input_matrix_wpn function, the output pos_mat size is 1x(HW)x3. Looking forward to your reply.

Zewei

**Comment by XuecaiHu** [comment]
Thank you for reporting this error. The dim of the input should be 3 not 6. I will correct it.
